src/overlays/actors/ovl_Bg_Haka_Tomb/z_bg_haka_tomb.c

```diff
@@ -9,7 +9,16 @@ void BgHakaTomb_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgHakaTomb_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgHakaTomb_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80BD6624(BgHakaTomb* this);
+void func_80BD66AC(BgHakaTomb* this, GlobalContext* globalCtx);
+void func_80BD6754(BgHakaTomb* this);
+void func_80BD6768(BgHakaTomb* this, GlobalContext* globalCtx);
+void BgHakaTomb_SetupDoNothing(BgHakaTomb* this);
+void BgHakaTomb_DoNothing(BgHakaTomb* this, GlobalContext* globalCtx);
+
+extern FlexSkeletonHeader D_06000EE8[];
```

Comment:
```suggestion
extern CollisionHeader D_06000EE8[];
```
This should fix the extra warning that was introduced and pass Jenkins